cloud/deployment/deployment_variable.go

```diff
@@ -114,23 +122,23 @@ func VariableModify(deploymentID, variableKey, variableValue, ws, envFile, deplo
 		newEnvironmentVariables = addVariablesFromFile(envFile, oldKeyList, oldEnvironmentVariables, newEnvironmentVariables, updateVars, makeSecret)
 	}
 
-	// create variable input
-	variablesCreateInput := astro.EnvironmentVariablesInput{
-		DeploymentID:         currentDeployment.ID,
-		EnvironmentVariables: newEnvironmentVariables,
-	}
-
 	// update deployment
-	environmentVariablesObjects, err := client.ModifyDeploymentVariable(variablesCreateInput)
+	err = Update(currentDeployment.Id, "", "", "", "", "", "", "", "", "", "", "", "", "", 0, 0, []astroplatformcore.WorkerQueueRequest{}, []astroplatformcore.HybridWorkerQueueRequest{}, newEnvironmentVariables, false, coreClient, platformCoreClient)
```

Comment:
Not in this PR, but we need to improve the Update function with a proper input and validations. This request with s many input arguments looks bad and prone to errors.